neo/Plugins/Plugin.cs

```diff
@@ -120,5 +125,26 @@ public static bool SendMessage(object message)
                     return true;
             return false;
         }
+
+        private static Assembly CurrentDomain_AssemblyResolve(object sender, ResolveEventArgs args)
+        {
+            if (args.Name.Contains(".resources"))
+                return null;
+
+            Assembly assembly = AppDomain.CurrentDomain.GetAssemblies().FirstOrDefault(a => a.FullName == args.Name);
+            if (assembly != null)
+                return assembly;
+
+            string filename = args.Name.Split(',')[0] + ".dll";
```

Comment:
What format do you receive here in `args.Name`?

Reply:
It's receiving the full assembly name, e.g.
```MyAssembly, Version=1.2.3.4, Culture=neutral, PublicKeyToken=null```

Reply:
You can use `AssemblyName` to parse the full name of the assembly:

```c#
AssemblyName a = new AssemblyName(args.Name);
string fullname = a.FullName;
string name = a.Name;
```

Reply:
Neat, just made that change in https://github.com/hal0x2328/neo/commit/6160d4cf119e92911d7df1166309a5b25a4eb556